dev-tools/mage/target/integtest/integtest.go

```diff
@@ -89,3 +93,31 @@ func PythonIntegTest(ctx context.Context) error {
 		return devtools.PythonTest(devtools.DefaultPythonTestIntegrationArgs())
 	})
 }
+
+// PackageSystemTests packages the python system tests results
+func PackageSystemTests() error {
+	excludeds := []string{".ci", ".git", ".github", "vendor", "dev-tools"}
+
+	// include run as it's the directory we want to compress
+	systemTestsDir := fmt.Sprintf("build%[1]csystem-tests%[1]crun", os.PathSeparator)
```

Comment:
Nit. This is a bit cryptic, could `filepath.Join` be used instead?
```suggestion
	systemTestsDir := filepath.Join("build", "system-tests", "run")
```